src/components/Question.jsx

```diff
@@ -227,12 +227,10 @@ export default class Question extends React.Component {
                   onMouseEnter={this._onMouseEnterHandler}
                   onMouseLeave={this._onMouseLeaveHandler}
                 >
-                  <h6 className="d-inline" id={question["@id"]}>
```

Comment:
I am little afraid that removing `id` can ruin navigation to the questions

Reply:
In the older version, the red text color for the wizard was applied only on the first render. It has the same behavior now.

Reply:
I returned the id for the question